app4java/src/main/java/com/nyxei/pollingus/PollingusApplication.java

```diff
@@ -1,13 +1,37 @@
 package com.nyxei.pollingus;
 
-import org.springframework.boot.SpringApplication;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class PollingusApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(PollingusApplication.class, args);
```

Comment:
Main method role is not to launch request but to launch the whole program. You should have a better place/object/class to do this job.

Reply:
je comprend mais je n'arrive pas à dissocier la requête de la méthode... si tu peux nous donnez une architecture pour ça

Reply:
Pas d'avancée ici.

Reply:
problème résolu dans le nouveau PR que j'ai effectué